### PR TITLE
Fixed rock questions replaced by techno using add instead of addPlayer

### PR DESCRIPTION
**Pull request by Terencyl**

### CODE REVIEW

**Comment by Badjeck** [comment]
oui

**Review by Lotfihihi — COMMENTED** [review]
OK

**Review by Badjeck — APPROVED** [review]

File: src/game.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ export class Game {
             return 'Sports';
         if (this.playerPool.getCurrentPlayerPlaces() == 10)
             return 'Sports';
-        if (this.questions.getIsTechnoQuestionsEnabled())
+        if (this.questions.getIsTechnoQuestionsEnabled() == true)
```
Comment (Saiiga) [inline src/game.ts:82]: remove "== true"

File: src/game.ts [hunk 2]
```diff
@@ -29,34 +29,34 @@ export class Game {
...
+                this.console.log("The category is " + this.currentCategory());
+                this.questions.askQuestion(this.currentCategory());
+            } else {
+                this.console.log(this.playerPool.getCurrentPlayer() + " is not getting out of the penalty box");
+                this.playerPool.isGettingOutOfPenaltyBox = false;
+            }
+        } else {
+
+            this.playerPool.setCurrentPlayerPlaces(this.playerPool.getCurrentPlayerPlaces() + roll);
             if (this.playerPool.getCurrentPlayerPlaces() > 11) {
```
Comment (Saiiga) [inline src/game.ts:53]: create a function to factorise

**Review by Saiiga — CHANGES_REQUESTED** [review]

**Pull request merged**